docs/dev/openstack-howto.md

```diff
@@ -0,0 +1,93 @@
+# OpenStack HOWTO
+
+It's possible to launch an all-in-one - for test and development - cluster on OpenStack.
+
+## The ocp-doit scripts
+
+We've created a set of scripts that make it easier to setup an all-in-one OpenStack. The OpenStack cluster should work either on a VM or a BM node. We recommend the latter.
+
+The scripts can be found in the following [repo](https://github.com/imain/ocp-doit)
```

Comment:
This is also mentioned by @russellb in #904 so I think we need to rebase such that there's a clear split between the dev-facing ocp-doit stuff (which is mostly covered by this PR) and the user-facing docs (which are mostly covered by #904) - probably we shouldn't mention ocp-doit at all in the user-facing page?

Reply:
The dev-user split makes sense.  For dev docs, I think a pointer to the ocp-doit repo is enough though, since we have detailed docs maintained in that repo.